tests/unit/plugins/europeana/record.spec.js

```diff
@@ -86,4 +86,65 @@ describe('plugins/europeana/record', () => {
       });
     });
   });
+
+  describe('fieldLabel()', () => {
+    const availableFields = [
+      'dcTitle',
+      'dcType',
+      'edmCountry',
+      'dctermsCreated',
+      'edmDataProvider',
+      'edmRights',
+      'dcContributor',
+      'dcCreator',
+      'dcDescription'
+    ];
+
+    describe('looking up an available key', () => {
+      let key;
+      for (key in availableFields) {
```

Comment:
Use `for...of` here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Iterating_over_an_Array